packages/select/src/Select/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+// reset user agent styles for button element type
+@mixin button-reset {
```

Comment:
Is this not shared with the original button component?

Reply:
Yes, we don't want to share styles between packages :)

Reply:
I guess another bonus of the All-in-One lol

Reply:
Yessiree. Alt, we have some other way of applying reset styles. Perhaps we just export a stylesheet of resets.